packages/amplify-e2e-tests/src/__tests__/datastore-modelgen.test.ts

```diff
@@ -26,7 +30,7 @@ describe('data store modelgen tests', () => {
 
     await expect(generateModels(projRoot)).resolves.not.toThrow();
     updateApiSchema(projRoot, projName, schemaWithError);
-    await expect(generateModels(projRoot)).rejects.toThrowError();
+    await generateModelsWithUnknownTypeError(projRoot);
```

Comment:
This is to fix the promise not rejecting but resolving. It turns out all the codegen commands will catch the error but never re-throw them. Also the `AmplifyError` is not integrated yet in codegen repo